src/Search/SearchHandler.php

```diff
@@ -49,7 +60,8 @@ public function search(AdminInterface $admin, $term, $page = 0, $offset = 20)
             /** @var $filter FilterInterface */
             if ($filter->getOption('global_search', false)) {
                 $filter->setCondition(FilterInterface::CONDITION_OR);
-                $datagrid->setValue($filter->getFormName(), null, $term);
+                $filter->setOption('case_sensitive', $this->caseSensitive);
+                $datagrid->setValue($filter->getFormName(), null, $this->caseSensitive ? $term : mb_strtolower($term));
```

Comment:
Wheb ``$this->caseSensitive`` is false ``mb_strtolower($term)`` will be apply to all values?

Reply:
mb_strtolower will apply for other filters where is no need.
If I use custom filter where I am ignore `case_sensitive` option I can't get original value.

Why you don't  move call `mb_strtolower` to StringFilter?

Reply:
sounds reasonable

Reply:
You mean sth like this:
```php
        if (ChoiceType::TYPE_EQUAL == $data['type']) {
            $queryBuilder->setParameter($parameterName, $data['value']);
        } else {
            $queryBuilder->setParameter($parameterName,
                sprintf(
                    $this->getOption('format'),
                    $this->getOption('case_sensitive') ? $data['value'] : mb_strtolower($data['value'])
                )
            );
        }
```

Reply:
Yes, like this.

Reply:
Done!

Reply:
Thanks